core/src/main/java/org/nervos/ckb/utils/address/AddressGenerator.java

```diff
@@ -91,7 +90,7 @@ public static String generateBech32mFullAddress(Network network, Script script)
         Numeric.hexStringToByteArray(
             TYPE_FULL_WITH_BECH32M
                 + Numeric.toHexStringNoPrefix(script.codeHash)
-                + Serializer.serializeHashType(script.hashType)
+                + String.format("%02x", script.hashType.pack() & 0xFF)
```

Comment:
the return type of `pack()` is a `byte` already, no need to do byte `&` with 0xFF
```suggestion
                + String.format("%02x", script.hashType.pack())
```

Reply:
This part is the old way to parse address. I didn't take consideration into them in refactor before I plan to replace with a new class to parse in the future.